### PR TITLE
Subscribe ui fixes

### PR DESCRIPTION
**Pull request by mwasilew**
Fixes: #723

### CODE REVIEW

File: squad/frontend/templates/squad/project.jinja2 [hunk 1]
```diff
@@ -25,3 +25,7 @@
 
 {% endwith %}
 {% endblock %}
+
+{% block javascript %}
+<script type="module" src='{{static("squad/project.js")}}'></script>
```
Comment (chaws) [inline squad/frontend/templates/squad/project.jinja2:30]: This file doesn't exist. Did you mean to include in this PR?

File: squad/frontend/templates/squad/_subscribe.jinja2 [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  		<form class="pull-right" action="{{url("settings-subscriptions")}}" method="POST">
+  		{% endif %}
+    		{{ csrf_input }}
+    		<input type="hidden" name="notification-strategy" value="all"/>
+    		<input type="hidden" name="email" value="{{ user.email }}"/>
+    		<input type="hidden" name="subscription" value="{{ project.id }}"/>
+    		{% if project.is_subscribed(user) %}
+    		<button class="btn btn-default" type="submit"><i class='fa fa-bell-slash'>&nbsp;</i></button>
+    		{% else %}
+    		<button class="btn btn-default" type="submit"><i class='fa fa-bell'>&nbsp;</i></button>
```
Comment (chaws) [inline squad/frontend/templates/squad/_subscribe.jinja2:14]: Nice! Looks really cool

**Comment by chaws** [comment]
Looks good to me. I'll wait for comments about `project.js` file then I can merge it

File: squad/frontend/templates/squad/project.jinja2 (hunk 1 above)
Comment (mwasilew) [reply squad/frontend/templates/squad/project.jinja2:30]: ah, that's leftover from AJAX implementation that I abandoned. Will remove.

**Comment by chaws** [comment]
Tests failed, I checked and it seems like Flake8 is now catching use of variables that are "hard to read": https://www.python.org/dev/peps/pep-0008/#names-to-avoid

Never heard of that before...

**Comment by mwasilew** [comment]
yeah, and it's in the test I didn't touch in this patch. Let me try to fix it.

**Comment by pfalcon** [comment]
Guys, I really appreciate a quick turnaround on the reports. But sometimes "quick" means "less attention to detail". And in #723, I specifically mentioned:

> Specifically, I might suggest to include "Subscription" tab in Project settings, where all settings are collected, but also add a quick-access straight from the project page(s), which can be in the form of a tab heading or a graphic icon (a bell or similar). **In both cases, it should lead to a page with additional subscription options and confirmation button, not lead to an immediate action with some "defaults".**

While, code like: https://github.com/Linaro/squad/pull/726/commits/fc0a4dea733ac7305bfb06b5046b5c496c77bb20#diff-b639dad4741e889c2d1ae340bc082f55R11 (i.e. POST + `&lt;input type="hidden" name="notification-strategy"`) makes me believe that a button was added which silently subscribes a user to "all" notifications.

This is a good improvement in one area (access to subscriptions), but arguably only a regression in another area - customizability of subscriptions. Because now a user would have lesser chance to discover that SQUAD allows different types ("strategies") of notifications, and could easily grow a misconception that SQUAD always sends unconditional notifications regardless of test status.

And some users are not interested in "all" notifications. Moreover, it's LAVA's problem that it offers only as "all" notifications, and SQUAD is advertized as a solution to that. It would be only sad that the way to leverage that becomes only harder to discover and leverage.

So again, I appreciate quick response, but IMHO, a more detailed implementation, even if longer timeframe, would be beneficial. Thanks!

**Pull request merged**